pages/my-page/components/VoteNotification/index.tsx

```diff
@@ -0,0 +1,39 @@
+import Image from "next/image";
+
+import classNames from "classnames/bind";
+import dayjs from "dayjs";
+
+import { VoteNotificationType } from "@/types/type";
+
+import styles from "./VoteNotification.module.scss";
+
+dayjs.locale("ko");
+
+const cn = classNames.bind(styles);
+
+interface VoteNotificationProps {
+  notification: VoteNotificationType;
+}
+
+export default function VoteNotification({ notification }: VoteNotificationProps) {
+  const endDate = dayjs(notification.localDateTime).format("YY.MM.DD (ddd)");
+
+  return (
+    <div className={cn("container", { isVoted: notification.voteParticipateType })}>
+      <div className={cn("label-container")}>
+        <div className={cn("left-labels")}>
+          <div className={cn("group-label")}>{notification.crewName}</div>
+          {notification.isChecked || <div className={cn("new-label")}>NEW</div>}
```

Comment:
```suggestion
          {notification.isChecked || <div className={cn("new-label")}>NEW</div>}
```
P5: `||` 이렇게 사용하면 `isChecked`와 관계없이 계속 렌더링되지 않나요??(사실 이렇게 안해봐서 잘 모르겠네요...)

Reply:
`isChecked` 값의 타입이 `boolean`이기 때문에, `isChecked`가 `true`일 경우에는 뒤쪽 `<div>` 태그는 렌더링 되지 않습니다!

Reply:
이건 제 개인적인 생각인데 그러면 `&&` 연산자를 사용하는게 더 의도 파악하기 쉽지않나요??
` { !notification.isChecked && <div className={cn("new-label")}>NEW</div> }`

Reply:
저도 예전에는 우혁님께서 말씀해주신대로 
```ts
{ !notification.isChecked && <div className={cn("new-label")}>NEW</div> }
``` 
이렇게 코드를 작성했었는데, 

이전 파트 때 멘토님께서 이 방식보다는 
```ts
{notification.isChecked || <div className={cn("new-label")}>NEW</div>}
```
이렇게 작성해주는 게 더 좋을 것 같다고 피드백 주신 적이 있었어서 바꿔봤습니다 🐰

Reply:
저는 오히려 의도 파악은 후자가 더 좋은 것 같아요 👀

Reply:
오 그렇군요 그 때 멘토님이 제시하신 근거는 어떤게 있었나요??

Reply:
특별한 근거가 있었던 건 아니고, 이렇게 작성하는게 더 일반적이다 정도의 이유였던 걸로 기억해요!

Reply:
앗 그런가요.. 남주님이 이전 멘토님한테 설득 당하셔서 바꾼줄 알았는데 그냥 저게 일반적인 방식이였군요!